### PR TITLE
sc_*, scamper: add German translation

### PR DESCRIPTION
**Pull request by marek22k**
sc_analysis_dump, sc_tracediff, sc_ttlexp, sc_warts2csv, sc_warts2json, sc_warts2pcap, sc_warts2text, sc_wartscat, sc_wartsdump, sc_wartsfilter, sc_wartsfix, scamper: add German translation

- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):*20240122*

### CODE REVIEW

File: pages.de/common/sc_analysis_dump.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# sc_analysis_dump
+
+> Ausgabe der traceroute-Daten in einem leicht zu parsenden Format.
```
Comment (gutjuri) [inline pages.de/common/sc_analysis_dump.md:3]:
```suggestion
> Ausgabe von Traceroute-Pfaden in einem leicht zu parsenden Format.
```

File: pages.de/common/sc_wartscat.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# sc_wartscat
+
+> Verkettet `warts`-Dateien.
```
Comment (gutjuri) [inline pages.de/common/sc_wartscat.md:3]:
```suggestion
> Füge mehrere `warts`-Dateien zusammen.
```

File: pages.de/common/sc_wartsdump.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# sc_wartsdump
+
+> Ausführlicher Ausgabe der in einer `warts`-Datei enthaltenen Daten.
```
Comment (gutjuri) [inline pages.de/common/sc_wartsdump.md:3]:
```suggestion
> Ausführliche Ausgabe der in einer `warts`-Datei enthaltenen Daten.
```

File: pages.de/common/sc_wartsfilter.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# sc_wartsfilter
+
+> Filtert bestimmte Datensätze aus einer `warts`-Datei.
+> Weitere Informationen: <https://www.caida.org/catalog/software/scamper/>.
+
+- Filtere alle Datensätze, welche eine bestimmten Ziel haben und schreibe sie in eine seperate Datei:
```
Comment (gutjuri) [inline pages.de/common/sc_wartsfilter.md:6]:
```suggestion
- Filtere alle Datensätze, welche ein bestimmtes Ziel haben und schreibe sie in eine separate Datei:
```

File: pages.de/common/sc_wartsfilter.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+`sc_wartsfilter -i {{path/to/input.warts}} -o {{path/to/output.warts}} -a {{192.0.2.5}} -a {{192.0.2.6}}`
+
+- Filtere alle Datensätze, welche ein Ziel in einem bestimmten Prefix haben und schreibe sie in eine seperate Datei:
```
Comment (gutjuri) [inline pages.de/common/sc_wartsfilter.md:10]:
```suggestion
- Filtere alle Datensätze, welche ein Ziel in einem bestimmten Prefix haben und schreibe sie in eine separate Datei:
```

File: pages.de/common/sc_wartsfix.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# sc_wartsfix
+
+> Rettet beschädigte `warts`-Dateien.
+> Weitere Informationen: <https://www.caida.org/catalog/software/scamper/>.
+
+- Speicher alle Datensätze (in einer separaten Datei) bis zum letzten intakten Datensatz:
```
Comment (gutjuri) [inline pages.de/common/sc_wartsfix.md:6]:
```suggestion
- Speichere alle Datensätze (in einer separaten Datei) bis zum letzten intakten Datensatz:
```

File: pages.de/common/sc_warts2text.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# sc_warts2text
+
+> Einfache Ausgabe der in einer `warts`-Datei enthaltenen Informationen.
+> Weitere Informationen: <https://www.caida.org/catalog/software/scamper/>.
+
+- Gebe die Informationen in einer `warts`-Datei als Text aus:
```
Comment (gutjuri) [inline pages.de/common/sc_warts2text.md:6]:
```suggestion
- Gib die Informationen in einer `warts`-Datei als Text aus:
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have some suggestions.

**Comment by marek22k** [comment]
@gutjuri Thanks for the hints!

@kbdharun Sorry for the late response. To be honest, I simply forgot about it.

**Review by gutjuri — APPROVED** [review]
LGTM! Thanks again for your contribution!

**Review by kbdharun — APPROVED** [review]

**Pull request merged**